indico/htdocs/js/indico/modules/reviews.js

```diff
@@ -53,7 +53,10 @@
                 $item.addClass('header-indicator-left');
             }
             if ($item.data('no-comment') !== undefined) {
-                $item.addClass('header-only');
+                var $ratingsDetails = $item.find('.ratings-details');
+                if (!$ratingsDetails.length || $ratingsDetails.css('display') === 'none') {
```

Comment:
`!$ratingsDetails.is(':visible')` doesn't work for the second check?

Reply:
works, works ;)